examples/making-an-eval-command.md

```diff
@@ -149,30 +149,7 @@ Be sure to supply the client as an input when we call the function later in the
     // ...
 ```
 
-* **For NodeJS versions 12.x through 14.x** (For NodeJS 15.x+, see next bullet)
-
-```javascript
-// If you are on NodeJS versions 12.x-14.x, in
-// order to replace all instances of our input
-// we need to add this function to the file (it
-// can be defined in the same area as the clean()
-// function).
-function replaceAll(haystack, needle, replacement) {
-  return haystack.split(needle).join(replacement)
-}
-
-const clean = async (client, text) => { 
-  // ...
-
-  // Then you will need to place this inside the
-  // clean function, before the result is returned.
-  text = replaceAll(text, client.token, "[REDACTED]");
-
-  // ...
-}
-```
-
-* **For Node 15.x and higher**, you do not need to define the [`replaceAll`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll) function, as it is native after Node v15 as a string prototype. A slight modification to the code will be required:
+* **For Node 15.x and higher**, you do not need to define the [`replaceAll`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll) function, as it is native after Node v15 as a string prototype.
```

Comment:
Is their a reason to have this at all? Discord.js v13 needs node v16+ and as such we don't need to explain why we use a method.

Reply:
I thought I removed that. Nevermind, I'll just fix it.